Storage/Storage/Model/OrderAttributionInfo+CoreDataProperties.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import CoreData
+
+
+extension OrderAttributionInfo {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<OrderAttributionInfo> {
+        return NSFetchRequest<OrderAttributionInfo>(entityName: "OrderAttributionInfo")
+    }
+
+    @NSManaged public var sourceType: String?
+    @NSManaged public var campaign: String?
+    @NSManaged public var source: String?
+    @NSManaged public var medium: String?
```

Comment:
Nit: we should keep the prefix UTM to clarify what these properties are.
```suggestion
    @NSManaged public var utmCampaign: String?
    @NSManaged public var utmSource: String?
    @NSManaged public var utmMedium: String?
```

Reply:
> In the screenshot, you have a UTM source. I propose not to use UTM as part of the label. It is technically correct that this data originates from UTM parameters, but we may expand our functionality and take other factors/sources into consideration. So, instead of UTM Source, I propose calling it Source.

Source - pe5sF9-2sD-p2#comment-3379

We should avoid adding `utm` for the above quoted reason.

Reply:
Interesting! Thanks for explaining.